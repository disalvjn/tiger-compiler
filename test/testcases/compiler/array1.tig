let type intarray = array of int
    var a := intarray [10] of 5
in for i := 0 to 10-1 do printInt(a[i]);
   for i := 0 to 10-1 do a[i] := i;
   for i := 0 to 10-1 do printInt(a[i])
end
