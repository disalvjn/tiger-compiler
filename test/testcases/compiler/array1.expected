55555555550123456789
